app/screens/contacts/Contacts.js

```diff
@@ -42,19 +42,17 @@ class Contacts extends Component<{}, State> {
   render() {
     const { addressToAdd } = this.state;
     return (
-      <ErrorBoundary>
-        <Wrapper>
-          <Header>My Contacts</Header>
-          <BodyWrapper>
-            <LeftPane>
-              <AllContacts addContact={({ address }) => this.setState({ addressToAdd: address })} />
-            </LeftPane>
-            <AddNewContact defaultAddress={addressToAdd} onSave={() => this.setState({ addressToAdd: '' })} />
-          </BodyWrapper>
-        </Wrapper>
-      </ErrorBoundary>
+      <Wrapper>
+        <Header>My Contacts</Header>
+        <BodyWrapper>
+          <LeftPane>
+            <AllContacts addContact={({ address }) => this.setState({ addressToAdd: address })} />
+          </LeftPane>
+          <AddNewContact defaultAddress={addressToAdd} onSave={() => this.setState({ addressToAdd: '' })} />
+        </BodyWrapper>
+      </Wrapper>
     );
   }
 }
 
-export default Contacts;
+export default ScreenErrorBoundary(Contacts);
```

Comment:
change this and following to 
```
Contacts = ScreenErrorBoundary(Contacts);
export default Contacts;
```
this will keep the component name for error stack

Reply:
OK.